src/contour/contour.yml

```
@@ -220,6 +220,11 @@ profiles:
             # whether or not to hide the scrollbar when in alt-screen.
             hide_in_alt_screen: true
 
+        # mouse setting
+        mouse:
+            # whether or not to hide mouse when typing
```

```suggestion
            # whether or not to hide mouse when typing
            #
            # Default value: true
```
I think it's worth mentioning the default (or simply just to stay consistent with most of the options) :)